src/main/webapp/dev/js/main/instructorStudentList.es6

```diff
@@ -96,9 +96,10 @@ function filterSection() {
  */
 function filterTeam() {
     $('input[id^="team_check"]').each(function () {
-        const courseIdx = $(this).attr('id').split('-')[1];
-        const sectionIdx = $(this).attr('id').split('-')[2];
-        const teamIdx = $(this).attr('id').split('-')[3];
+        const $this = $(this);
```

Comment:
It looks like there is no other usage of `$(this)` other than the next three lines. You can make `$(this).attr('id').split('-')` the variable instead.